src/components/HomepageBanner.js

```diff
@@ -0,0 +1,301 @@
+import React from 'react';
+import { css } from '@emotion/react';
+import { Button, Link } from '@newrelic/gatsby-theme-newrelic';
+import bannerForeground from '../images/bannerForeground.svg';
+import bannerForegroundMobile from '../images/bannerForegroundMobile.svg';
+import relicsAtWork from '../images/photo-relics-at-work-01.png';
+import relicsAtWorkMobile from '../images/relics-at-work--SM.png';
+
+const BREAKPOINTS = {
+  NO_LEFT_NAV: '760px',
+  SINGLE_COLUMN: '650px',
+  MOBILE: '450px',
+};
+
+const HomepageBanner = () => {
+  return (
+    <div
+      css={css`
+        max-height: 423px;
+        position: relative;
+        overflow: hidden;
+        border-radius: 0.5rem;
+        padding-top: calc(423 / 1252 * 100%);
+        background: var(--color-brand-700);
+
+        @media screen and (max-width: ${BREAKPOINTS.SINGLE_COLUMN}) {
+          padding-top: calc(450 / 380 * 100%);
+          max-height: 450px;
+        }
+
+        @media screen and (max-width: ${BREAKPOINTS.MOBILE}) {
+          padding-top: 0;
+          max-height: none;
+          height: 500px;
+        }
+        @media screen and (max-width: 320px) {
+          height: 460px;
+        }
+      `}
+    >
+      <div
+        css={css`
+          position: absolute;
+          top: 0;
+          left: 0;
+          width: 100%;
+          height: 100%;
+          @media screen and (max-width: ${BREAKPOINTS.SINGLE_COLUMN}) {
+            display: flex;
+            flex-direction: column;
+          }
+        `}
+      >
+        <picture>
+          <source
+            srcSet={bannerForegroundMobile}
+            media={`(max-width:${BREAKPOINTS.SINGLE_COLUMN})`}
+          />
+          <img
+            src={bannerForeground}
+            alt="banner foreground"
+            css={css`
+              position: relative;
+              height: 100%;
```

Comment:
This _might_ solve the issue with the little sliver of background showing through.

```suggestion
              height: calc(100% + 1px);
```